specgen_connector_migrator/internal/connector_go.go

```diff
@@ -0,0 +1,84 @@
+// Copyright © 2024 Meroxa, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package internal
+
+import (
+	"fmt"
+	"os"
+	"regexp"
+	"strings"
+)
+
+type ConnectorGoMigrator struct {
+}
+
+func (a ConnectorGoMigrator) Migrate(workingDir string) error {
+	connectorGoPath, connectorGo, err := readFile(workingDir, "connector.go")
+	if err != nil {
+		return err
+	}
+
+	updatedConnectorGo := strings.ReplaceAll(
+		connectorGo,
+		`// limitations under the License.`,
+		`// limitations under the License.
+
+//go:generate specgen`,
```

Comment:
Reminder to change this once https://github.com/ConduitIO/conduit-connector-sdk/pull/246 is merged.

Reply:
@hariso looks like https://github.com/ConduitIO/conduit-connector-sdk/pull/246 was already merged. Good to go then?